mobile/lib/models/database.dart

```diff
@@ -0,0 +1,101 @@
+import 'dart:io';
+
+import 'package:app/models/enum_constants.dart';
+import 'package:app/utils/utils.dart';
+import 'package:drift/drift.dart';
+import 'package:drift/native.dart';
+import 'package:path/path.dart' as p;
+import 'package:path_provider/path_provider.dart';
+
+part 'database.g.dart';
+
+abstract class Insight extends Table {
+  DateTimeColumn get time => dateTime()();
+  RealColumn get pm2_5 => real()();
+  RealColumn get pm10 => real()();
+  BoolColumn get available => boolean().withDefault(const Constant(true))();
+  TextColumn get siteId => text()();
+  IntColumn get frequency => intEnum<Frequency>()();
+
+  @override
+  Set<Column> get primaryKey => {
+        siteId,
+        frequency,
+        time,
+      };
+}
```

Comment:
Insight (measurements) model

Reply:
Thanks @NoahNsimbe for this. I will just leave a few file organisational questions for your review.

I sense that the insights.dart file has been deleted in favor of the database.dart file.

1. What is the main motivation for this rename/restructure?
2. What is the organisation structure here? Is the `database` file supposed to represent an entity?